node/service/src/rpc/runtime_info.rs

```diff
@@ -0,0 +1,94 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2023 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! This module is used to add custom runtime irrelevant RPC endpoints to the node.
+
+use jsonrpsee::{
+    core::{Error as RpcError, RpcResult},
+    proc_macros::rpc,
+};
+use sc_client_api::StorageProvider;
+use sc_executor_common::runtime_blob::RuntimeBlob;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::traits::Block as BlockT;
+use std::{marker::PhantomData, sync::Arc};
+
+#[rpc(server)]
+pub(crate) trait RuntimeInfo<BlockHash> {
+    #[method(name = "gear_wasmRuntimeVersion")]
```

Comment:
Probably, it is worth renaming this into something like
`runtime_wasmVersion`
`runtime_wasmRuntimeVersion`
any other options are welcome

Reply:
RuntimeVersion refers to https://github.com/gear-tech/gear/blob/4c345bc4270982cad4a695d1b910c6277f4498b6/runtime/gear/src/lib.rs#L106C1-L107 so it isn't good for me

Reply:
furthermore, it's not just `gear_` stuff so `runtime_` prefix is preferable

Reply:
maybe `runtime_rev`/`runtime_revision`/`runtime_blobRev`?